R/FilterState.R

```diff
@@ -1066,19 +1070,35 @@ LogicalFilterState <- R6::R6Class( # nolint
     #'  id of shiny element
     ui = function(id) {
       ns <- NS(id)
-      fluidRow(
+      l_counts <- as.numeric(names(private$choices))
+      is_na_l_counts <- is.na(l_counts)
+      if (any(is_na_l_counts)) l_counts[is_na_l_counts] <- 0
+      labels <- lapply(seq_along(private$choices), function(i) {
+        l_count <- l_counts[i]
+        l_freq <- l_count / sum(l_counts)
+        if (is.na(l_freq) || is.nan(l_freq)) l_freq <- 0
         div(
-          class = "relative",
-          # same overlay as for choices with no more than (default: 5) elements
-          div(
-            class = "filterPlotOverlayBoxes",
-            plotOutput(ns("plot"), height = "100%")
-          ),
+          class = "choices_state_label",
+          style = sprintf("width:%s%%", l_freq * 100),
+          span(
+            class = "choices_state_label_text",
+            sprintf(
+              "%s (%s)",
+              private$choices[i],
+              l_count
+            )
+          )
+        )
```

Comment:
this can be probably defined as a function in `utils` as it is called twice.

Reply:
It is not so easy as these are 2 separate classes and we could not add a private method to the base class as it make no sense. A utils function is not a general for me too.